v3ioutils/schema.go

```diff
@@ -154,6 +155,11 @@ func (s *OldV3ioSchema) merge(new *OldV3ioSchema) (bool, error) {
 		changed = true
 	}
 
+	if s.SortingKey != new.SortingKey && new.SortingKey != "" {
```

Comment:
What if i had a table with sorting key `colA` then writing a new DF without a sorting key ? looks like you override the schema with an empty sortingKey which will corrupt the data. you should return an error in that case.
I did some changes for that in my PR probably should use that 😄